### PR TITLE
Binding API modification adaptation to implement application failover

### PR DESCRIPTION
**Pull request by Poor12**
**What type of PR is this?**
/kind api-change

**What this PR does / why we need it**:
Binding API modification adaptation to implement application failover

**Which issue(s) this PR fixes**:
Part of #3169 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
None

### CODE REVIEW

File: pkg/apis/work/v1alpha2/binding_types.go [hunk 1]
```diff
@@ -275,6 +292,36 @@ type AggregatedStatusItem struct {
...
+	// CreationTimestamp is a timestamp representing the server time when this AggregatedStatusItem was
+	// created.
+	// It represents the timestamp when the control plane first collects status of the resource running in a member cluster.
+	//
+	// Clients should not set this value to avoid the time inconsistency issue.
+	// It is represented in RFC3339 form(like '2021-04-25T10:02:10Z') and is in UTC.
+	//
+	// Populated by the system. Read-only.
+	// +optional
+	CreationTimestamp metav1.Time `json:"creationTimestamp,omitempty"`
```
Comment (Poor12) [inline pkg/apis/work/v1alpha2/binding_types.go:305]: I wonder if `CreationTimestamp` is too common.

**Review by RainbowMango — COMMENTED** [review]
/assign

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**